src/SFML/Audio/SoundBufferRecorder.cpp

```diff
@@ -70,9 +70,10 @@ const sfSoundBuffer* sfSoundBufferRecorder_getBuffer(const sfSoundBufferRecorder
 {
     CSFML_CHECK_RETURN(soundBufferRecorder, nullptr);
 
-    soundBufferRecorder->SoundBuffer.This = soundBufferRecorder->This.getBuffer();
+    soundBufferRecorder->SoundBuffer->This = soundBufferRecorder->This.getBuffer();
 
-    return &soundBufferRecorder->SoundBuffer;
+    assert(soundBufferRecorder->SoundBuffer && "sfSoundBufferRecorder_getBuffer() Cannot access null sound buffer");
```

Comment:
Can we, should do assert? It's the first of its kind (would also require an header include).

Reply:
I'm not 100% what I wrote is correct. I need to look at it some more.

Reply:
I changed this to what Kimci recommended and think it's correct now.